corehq/apps/events/models.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+from datetime import date
+from typing import Literal
+from django.utils.translation import gettext_lazy as _
+
+from corehq.form_processor.models import CommCareCase
+from corehq.apps.users.models import WebUser
+from corehq.apps.events.utils import create_case_with_case_type
+from corehq.sql_db.util import get_db_aliases_for_partitioned_query
+from corehq.apps.events.exceptions import EventNotPersistedError
+
+
+EVENT_CASE_TYPE = 'commcare-event'
+
+AttendeeListStatus = Literal[
+    'Not started',
+    'In progress',
+    'Under review',
+    'Rejected',
+    'Accepted',
+]
+ATTENDEE_LIST_STATUS_CHOICES = [
+    ('Not started', _('Not started')),
+    ('In progress', _('In progress')),
+    ('Under review', _('Under review')),
+    ('Rejected', _('Rejected')),
+    ('Accepted', _('Accepted')),
+]
+
+EVENT_CASE_PROPERTIES = [
+    'start_date',
+    'end_date',
+    'attendance_target',
+    'sameday_reg',
+    'track_each_day',
+    'is_open',
+    'attendee_list_status',
+]
+
+
+class Event:
```

Comment:
Probably simpler to use a normal Django model with Django model fields. ([See note in spec](https://docs.google.com/document/d/1hU2SQvnfkfxu3RoxhRCKL6IuWl6zO7nuZFEbgF5grrE/edit#heading=h.2533fy976nfz).)

Reply:
See [commit](https://github.com/dimagi/commcare-hq/pull/32630/commits/ded1dadfcd530b6efe269ab34981d2aadb7b7d20)